packages/flutter/test/material/ink_paint_test.dart

```diff
@@ -384,11 +384,11 @@ void main() {
     await tester.pumpAndSettle();
   });
 
-  testWidgets('Cancel an InkRipple that was disposed when its animation ended', (WidgetTester tester) async {
+  testWidgets('Cancel an InkRipple that was disposed before its animation started', (WidgetTester tester) async {
```

Comment:
@HansMuller  This test case was added by you in https://github.com/flutter/flutter/pull/17787, but the name is probably wrong since it's the same as [the previous test case](https://github.com/flutter/flutter/pull/81445/files#diff-cac344a770100da2307c9634b9b4c3986735f126606178967b732a11d43c0447R354). Can you check if the new name I came up makes sense?

Reply:
The new description looks good.